recipe/meta.yaml

```
@@ -1,16 +1,16 @@
 {% set name = "snowpark-checkpoints" %}
-{% set version = "0.2.0" %}
+{% set version = "0.2.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/snowflakedb/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: f1b82f1bc326d793347d06f164f2e4371ebb7050c8f52b6cc4fbf0824450ccc8
+  sha256: 7112de4a13a8d34e399c0959b716c95ab31d988cf96c0dbf6216dd9e7a5d45a5
 
 build:
-  # https://github.com/snowflakedb/snowpark-checkpoints/blob/v0.1.4/pyproject.toml#L15
+  # https://github.com/snowflakedb/snowpark-checkpoints/blob/v0.2.1/pyproject.toml#L15
   # From the upstream repository: requires-python = '>=3.9,<3.12'
   skip: True  # [py<39 or py>311 or s390x]
```

```suggestion
  skip: True  # [py<39 or py>312 or s390x]
```
https://github.com/snowflakedb/snowpark-checkpoints/blob/v0.2.1/pyproject.toml#L15

---

py<312 included, so everything is correct.

---

skiped py>311

---

I need new glasses!